data/alcb1310.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Andres Court",
+  "type": "personal",
+  "bio": "Full-stack developer with a passion for photography",
+  "avatar": "https://github.com/alcb1310.png",
```

Comment:
We no longer need the `avatar` key. It will automatically pick your GitHub Profile. So please remove this from your `json`.

```suggestion
```

Reply:
Updated the file as suggested